Implementacija/savetovaliste/app/Views/odgovori.php

```diff
@@ -44,286 +32,324 @@
-            </div>
-            <br>
-
-          </div>
-
-
-          <!-- DUGMAD -->
-          <div class="input ">
-            <div id="like">
-              <div style="float: right;">
-                <button onclick="window.location.href='pocetna_stranica.html';" class="w3-button buttons" style="font-weight: normal;">
-                    Odustani</button> &nbsp
-                <button  class="w3-button buttons"><u onclick="alert('Odgovor je poslat!')" style="text-decoration: none; font-weight: normal;">Pošalji</u></button>
-              </div>
-            </div>
-          </div>
-          <br>
-        </div>
-      </form>
-
-        <!-- KRAJ UNOSI -->
-      </div>
-
-      <br>
-      <!-- LITERATURA -->
-      <div class="w3-col l4">
-        <div class="w3-white w3-margin w3-card-2">
-          <div class="w3-container w3-padding gradient_literature letters">
-            <h4>Preporučena literatura za sve ljubitelje psihologije</h4>
-          </div>
-          <ul class="w3-ul w3-hoverable w3-white">
-
-            <li>
-              <div class="w3-row">
-                <div class="w3-col s2 w3-white w3-center">
-                  <img src="photos/50_ideja_koje_bi_stvarno_trebalo_da_znate_psihologija-adrijan_fernam_s.jpg"
-                    style="width:100%;">
+    <div class="w3-col l8 s12">
+    <br>
+    
+    <?php 
+    use App\Models\KorisnikModel;
+    use App\Models\PitanjeModel;
+    use App\Models\OdgovorModel;
+    use App\Models\KorisnikOcenioOdgovorModel;
+    use App\Models\KorisnikOcenioPitanjeModel;
+        
+        if (count($odgovori)==0) {
+            $idPitanja=$pitanje->idPitanje;
+            $korisnikOcenioPitanjeModel=new KorisnikOcenioPitanjeModel();
+            $likes=$korisnikOcenioPitanjeModel->findNumOfLikes($idPitanja);
+            $dislikes=$korisnikOcenioPitanjeModel->findNumOfDislikes(($idPitanja));
+            
+            echo '
+            <div class="w3-container w3-light-grey w3-margin w3-padding-large w3-card-4">
+                <div class="w3-left" style="padding-left: 5%;"> <br> 
+                    <h3 class="letters_dark_blue"> <b> ' . $pitanje->naslovPitanja . '   </b>  </h3>
                 </div>
-                <div class="w3-col s10 w3-left letters_dark_blue" style="padding-left: 10px;">
-                  <span class="w3-medium"><b>50 ideja koje bi stvarno trebalo da znate - Adrijan Fernam</b></span> <br>
-                  <span style="font-weight: 500;"> Kolika je razlika između muškog i ženskog mozga? Postoji li zaista altruizam? Da li je naš um,
-                    odmah po rođenju, neispisana tablica?</span>
-                    
+            ';   
+            
+            if($pitanje->postavljenoAnonimno==1)
+            {
+                echo '
+                    <div class="w3-right" style="padding-right: 6%;"> <br>
+                        <h3 class="letters_dark_blue"><b> ' . "Anonimno" . '</b></h3>
+                    </div> ';
```

Comment:
Čisto da bi bio mnogo lepši i kraći kod, je l bi mogla ove zajedničke stvari da izdvojiš, tj. ono što se svaki put ispiše. Recimo ovako: 
```suggestion
                echo '<div class="w3-right" style="padding-right: 6%;"> <br>
                          <h3 class="letters_dark_blue"><b> ';
                // Ovde logika za ispisivanje imena OR "Anonimno"
                echo '</b></h3>
                          </div> ';
```

Reply:
Ako imas vremena i zelje, slobodno. Kod radi